### ISSUE TITLE
As an API user, I want to retrieve applications for a particular address

### ISSUE THREAD

**Issue by jamezpolley** [openaustralia/planningalerts#1417]
As an API user, I'm interested in the history of applications at a particular address. I use the "Single Location by Address" API call to retrieve results for that address.

However, this often returns lots of applications I'm not interested in, but the applications I *am* interested in don't appear in the results.

For instance, in a very active area, it's common for the most recent application at the address I'm interested in to be further down than the first page of results.

I can somewhat work around this by decreasing the search radius:
  * For example, looking at the address "14+Brook+Road+Glenbrook%2C+NSW+2773" the first page of results with the default radius (4000m) has no hits at that address
  * dropping the search radius to 150m finds 2 hits
  * dropping to 15m finds *only* those two.

**Describe the solution you'd like**

TLDR: Add support for "closest first" to the API calls, to match what's on the website.

The solution that makes the most sense to me would be to mirror the options on the website; https://www.planningalerts.org.au/?q=14+Brook+Rd%2C+Glenbrook+NSW+2773 shows "Most recent first", but https://www.planningalerts.org.au/?q=14+Brook+Rd%2C+Glenbrook+NSW+2773&sort=distance shows "closest first". This puts the results for my specific address right on top, and then the results at my immediate neighbours after that.

It seems as though this should be the simplest solution to implement, as it's already in place on the website.

**Describe alternatives you've considered**

* An alternative solution might be to have flag for "Just this specific address". However, I feel that this might be fragile in cases where addresses change slightly. For instance, if I search for "123A X Road", but all the applications have been listed as "123 X Road"; or if I search for "25-27 X Road" but all the applications were listed as "25 X Road".

* Another alternative might be to have a flag for "I'm really just interested in this address or right beside it", which simply sets the radius very small - say, 75m or 150m. This probably avoids most of the problems with matching a specific address, but may also be confusing if there's an application which has been geocoded as being 200m away and so doesn't make it onto the list. Simply sorting by "closest first" makes it more explicitly the API user's responsibility to make the decision about when to stop looking down the list.

**Additional context**
This feature has been requested by one commercial customer and would be useful for at least one more.

**Comment by jamezpolley** [comment]
Issues #1020 and #267 seem to be slightly related, in that they're focused on a particular address as well. However, they're both looking at the web UI, and #1020 at least seems to largely be addressed by the "Closest first" button in the UI. There's no equivalent of "Closest first" in the API

**Comment by jamezpolley** [comment]
* Start by dropping search radius
